src/pages/homeView.js

```diff
@@ -1,10 +1,32 @@
 import React from "react";
 import Button from "../components/global/Button";
+import HomepageImage from "../assets/images/HomepageImage";
+import DateTime from "../components/DateTime";
 
 function homeView() {
+  const getGreeting = () => {
+    const myDate = new Date();
+    const hrs = myDate.getHours();
+
+    if (hrs < 12) return "Good Morning";
+    else if (hrs >= 12 && hrs <= 17) return "Good Afternoon";
+    else if (hrs >= 17 && hrs <= 24) return "Good Evening";
```

Comment:
You don't need to check hrs >= 12 here.

It would be better if you just did

`return hrs < 12 ? "Good Morning" : hrs < 18 ? "Good Afternoon" : "Good Evening";`

or 

```
if (hrs < 12) {
 return "Good Morning";
} else if (hrs < 18) {
return "Good Afternoon"
} else {
return "Good Evening"
```